stdlib/platform/java/fileSystem/BoundedReader.wyt

```diff
@@ -0,0 +1,9 @@
+import wyvern.option
+type Option = option.Option
+
+resource type BoundedReader
+    effect Read
+    def read():{this.Read}Int
```

Comment:
This should return Option[Int], right?

Reply:
I think that at some point we discussed this for read/readLine, since for readLine it would make sense to return option.None instead of a String, since it wouldn't be able to distinguish between a line of the empty string and not having a new line to read, whereas for read it would just return -1 when there's nothing left to be read. I can still change it if you would prefer it to be an option for consistency.

Reply:
Oh, right.  read() is returning a character, right?  In that case -1 makes sense.